.gitignore

```diff
@@ -8,4 +8,7 @@ keys/
 docs/generated
 docs/README.rst
 protos/
+!remme/protos/.keep
+!remme/protos/__init__.py
+remme/protos/
```

Comment:
This one is a good point, intended to do this myself today :)
Also, if those files are ignored shouldn't we remove them from the repo and make a notice in README on compiling them before starting any work?

Reply:
Tried to do this, but when tests are executed, it does not want to load proto as module.